x-pack/test/alerting_api_integration/spaces_only/tests/actions/execute_unsecured_action.ts

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import type { SearchTotalHits } from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
+import { FtrProviderContext } from '../../../../common/ftr_provider_context';
+import { ObjectRemover } from '../../../common/lib';
+
+// eslint-disable-next-line import/no-default-export
+export default function createUnsecuredActionTests({ getService }: FtrProviderContext) {
+  const supertest = getService('supertest');
+  const kibanaServer = getService('kibanaServer');
+  const es = getService('es');
+  const retry = getService('retry');
+
+  describe('execute unsecured action', () => {
```

Comment:
since there is some namespace bits in this PR, seems like we should add a test for a non-default space as well ...

Reply:
Added in https://github.com/elastic/kibana/pull/178682/commits/c24ebb0c09a766b3f706135a525079ec4a46d0cc